app/bundles/FormBundle/Entity/Form.php

```diff
@@ -188,6 +195,11 @@ public static function loadMetadata (ORM\ClassMetadata $metadata)
             ->nullable()
             ->build();
 
+        $builder->createField('renderStyle', 'boolean')
+            ->columnName('renderStyle')
```

Comment:
Let's change this to render_style so that it follows suit with the other column formats.